### PR TITLE
[KOGITO-8276] Fix setup log

### PR DESCRIPTION
**Pull request by desmax74**
See: https://issues.redhat.com/browse/KOGITO-8276
The operator doesn't print nothing

**Description of the change:**

**Motivation for the change:**

**Checklist**

- [ ] Add or Modify a unit test for your change
- [ ] Have you verified that tall the tests are passing?

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Review by radtriste — APPROVED** [review]

File: main.go [hunk 1]
```diff
@@ -60,8 +62,15 @@ func main() {
...
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	opts := zap.Options{
+		Development: true,
+	}
+	opts.BindFlags(flag.CommandLine)
+
 	flag.Parse()
 
+	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
```
Comment (davidesalerno) [inline main.go:72]: Why are we using `zap` instead of `klog` as agreed?

Comment (davidesalerno) [reply main.go:72]:
```suggestion
	ctrl.SetLogger(klog.NewKlogr())
```

Comment (desmax74) [reply main.go]: Right fixed

**Review by manstis — COMMENTED** [review]
It's probable that the logger will need setting in the `container-builder` module too.

https://github.com/kiegroup/kogito-serverless-operator/blob/a337a7ef9a6717aa35fbb6a0d14e96be0c1b36de/container-builder/main.go

File: api/const.go [hunk 2]
```diff
@@ -19,4 +19,5 @@ package api
 const (
 	// ComponentName just a name to identify this package/component/application
 	ComponentName = "sonataflow"
+	LogName       = "sonataflow-operator"
```
Comment (ricardozanini) [inline api/const.go:22]:
```suggestion
	LogName       = "sonataflow-manager"
```

**Review by manstis — APPROVED** [review]
LGTM :+1:

File: config/manager/manager.yaml [hunk 3]
```diff
@@ -33,7 +33,7 @@ spec:
         - /usr/local/bin/manager
         args:
         - --leader-elect
-        - '-v=2'
+        - --v=0
```
Comment (ricardozanini) [inline config/manager/manager.yaml:36]: This must be `2`. What happened? The generator is enforcing it? 🤔

Comment (desmax74) [reply config/manager/manager.yaml:36]: @ricardozanini  Do you want INFO instead of ERROR as a default LOG level in the operator ?

**Review by ricardozanini — APPROVED** [review]

**Review by davidesalerno — APPROVED** [review]

**Pull request merged**